addons/account_edi/models/account_move.py

```diff
@@ -41,19 +44,32 @@ def _compute_edi_state(self):
             else:
                 move.edi_state = False
 
-    @api.depends('edi_document_ids.error')
-    def _compute_edi_error_count(self):
+    @api.depends('edi_document_ids.error', 'edi_document_ids.error_level')
+    def _compute_edi_error(self):
         for move in self:
-            move.edi_error_count = len(move.edi_document_ids.filtered(lambda d: d.error))
+            move.edi_blocked = 0
+
+            errors = move.edi_document_ids.filtered(lambda d: d.error)
+            levels = self.env['account.edi.document']._fields['error_level'].selection
+            strings = []
+            for level, string in levels:
+                err = len(errors.filtered(lambda d: d.error_level == level))
+                if err:
+                    strings.append(_('%s %s', err, string))
+                    if level == 'error':
+                        move.edi_blocked = err
+
+            move.edi_error_message = ', '.join(strings)
```

Comment:
It's difficult to figure out how it looks like. I was thinking about something:
```
<div>
  <span>CFDI EDI: </span><span class="info-alert">error alert panic run away!</span>
  ...
</div>
```

Reply:
This is actually done in the tree view. This is only the message appearing in the header ('Electronic Invoicing: 1 error, 2 warnings') => I renamed the field so it's more clear that it's the header.